pkg/server/server.go

```diff
@@ -130,6 +130,18 @@ func (y *Server) optionsSecret(w http.ResponseWriter, _ *http.Request) {
 	w.Header().Set("Access-Control-Allow-Headers", "content-type")
 }
 
+func (y *Server) configHandler(w http.ResponseWriter, r *http.Request) {
+	viper.SetEnvPrefix("yopass")
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
```

Comment:
```suggestion
```

Reply:
I'm a bit worried that this might cause issues for existing environment variables that are not prefixed. I had to revert a PR recently that caused issues so lets write the full env name further down.